eth/backend.go

```diff
@@ -525,7 +524,8 @@ func (s *Ethereum) BloomIndexer() *core.ChainIndexer   { return s.bloomIndexer }
 func (s *Ethereum) Protocols() []p2p.Protocol {
 	protos := eth.MakeProtocols((*ethHandler)(s.handler), s.networkID, s.ethDialCandidates)
 	if s.config.SnapshotCache > 0 {
```

Comment:
If we want to disable `SnapshotCache`, we should probably do that here: https://github.com/scroll-tech/go-ethereum/blob/zkrollup/eth/backend.go#L102